src/java.base/unix/native/libjli/java_md_solinux.h

```diff
@@ -60,4 +66,29 @@ static const char *user_dir     = "/java";
 #include <pthread.h>
 #endif
 
+#ifdef AIX
+#define ZLIBNX_PATH "/usr/opt/zlibNX/lib"
+
+#ifndef POWER_9
+#define POWER_9 0x20000 /* 9 class CPU */
+#endif
+
+#ifndef POWER_10
+#define POWER_10 0x40000 /* 10 class CPU */
+#endif
+
+#define __power_9_andup() ((POWER_9  == _system_configuration.implementation) \
```

Comment:
Identifiers containing two underscores are reserved by C (see [1]), I suggest you just remove the two leading underscores from `__power_9_andup` and `__power_nx_gzip`.

[1] https://en.wikipedia.org/wiki/Reserved_word

Reply:
These names are used in `systemcfg.h` and suppose to be defined here when not in the header. 
But given that now its always defined (not conditionally) I removed the underscores as suggested.